_posts/en/newsletters/2020-10-28-newsletter.md

```diff
@@ -0,0 +1,228 @@
+        is likely to make payment secrets required by default.  We would
+        welcome other implementations to do the same."
+
+        A second set of emails from [Riard][riard6] and
+        [Fromknecht][fromknecht6] describe this issue in detail.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why is the Bitcoin Core wallet database moving from Berkeley DB to SQLite?]({{bse}}99620)
+  Michael Folkson references Andrew Chow's [blog post][achow wallet blog post]
+  including pros of SQLite, cons of Berkeley DB, and a bigger picture [proposed
+  timeline][wallet proposed timeline] to migrate from legacy to
+  [descriptor][topic descriptors] wallets.
+
+- [What are Merklized Alternative Script Trees?]({{bse}}99539)
+  Murch and Michael Folkson describe the history of [MAST][topic mast]
+  discussions in Bitcoin, the differences between Merklized Abstract Syntax
+  Trees and Merklized Alternative Script Trees, and the relation to taproot's
+  [BIP341][] proposal.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update tuesday (or delete if no releases/RCs)
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+FIXME
+
+- [Bitcoin Core #20198][] Show name, format and if uses descriptors in bitcoin-wallet tool FIXME:dongcarl
+
+- [C-Lightning #4046][] libplugin: support for sending notifications.  FIXME:moneyball
+
+- [C-Lightning #4139][] updates the `multifundchannel` RPC with a new
+  `commitment_feerate` parameter that sets the initial feerate for
+  commitment transactions in the channel.  Previously, the same feerate
+  was used for both the funding transaction and initial commitment
+  transactions.  This new option allows users to pay a higher feerate to
+  get a channel open quickly and then commit to a lower feerate when
+  they're using [anchor outputs][topic anchor outputs] to allow fee
+  bumping commitment transactions or they have another reason to be
+  willing to pay low commitment feerates.
```

Comment:
This sentence feels a little awkward. Rearranged the placement of anchor outputs.
```suggestion
  get a channel open quickly and then commit to a lower feerate when
  they're willing to pay low commitment feerates to allow fee bumping
  commitment transactions such has when using
  [anchor outputs][topic anchor outputs].
```